docs/Animations.md

```diff
@@ -444,14 +445,14 @@ var App = React.createClass({
 
     return (
       <View style={styles.container}>
-        <TouchableWithoutFeedback onPressIn={this._onPressIn}
-                                  onPressOut={this._onPressOut}>
+        <TouchableWithoutFeedback onPressIn={() => this._onPressIn()}
+                                  onPressOut={() => this._onPressOut()}>
```

Comment:
Is this necessary? I thought that passing functions directly is fine, since React auto-binds `this`?

cc: @davidaurelio